assets/application.json

```
@@ -215,7 +215,23 @@
               "maxLength": 1000,
               "title": "Instruçoes de entrega",
               "description": "Insira informações adicionais para retirada ou entrega do pedido"
-            }
+            },
+            "product_ids": {
+              "title": "Lista de produtos selecionados",
+              "description": "Se preenchido, a condição será efetivada com qualquer item da lista no carrinho",
+              "type": "array",
+              "items": {
+                "type": "string",
+                "pattern": "^[a-f0-9]{24}$",
+                "title": "ID do produto"
+              }
+            },
+            "all_product_ids": {
+              "type": "boolean",
+              "title": "Ativar condição para todos itens da lista",
+              "description": "Se ativo, será aplicado a condição configurada apenas para a lista de produtos no carrinho",
```

melhorou bem, mas acho que dá pra ficar mais claro especialmente o segundo campo.

```suggestion
              "description": "Se preenchido, a regra de envio estará disponível apenas se pelo menos um dos itens estiver no carrinho",
              "type": "array",
              "items": {
                "type": "string",
                "pattern": "^[a-f0-9]{24}$",
                "title": "ID do produto"
              }
            },
            "all_product_ids": {
              "type": "boolean",
              "title": "Checar todos os produtos",
              "description": "Se ativo, a regra será disponibilizada apenas se todos os produtos selecionados estiverem no carrinho",
```